i6_models/parts/blstm.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass()
+class BlstmEncoderConfig(ModelConfiguration):
+    num_layers: int
+    """number of bi-directional LSTM layers, minimum 2"""
+    input_dimension: int
+    """input dimension size"""
+    hidden_dimension: int
+    """hidden dimension of one direction of LSTM, the total output size is twice of this"""
+    dropout: float
+    """nn.LSTM supports an internal Dropout layer"""
+    enforce_sorted: bool = True
```

Comment:
I think a more descriptive name for this would be better. E.g. what this does, what his is about (seq lengths).

Reply:
I would rather keep the actual name of the parameter.